tools/src/commands/PublishProjectTemplates.ts

```diff
@@ -126,31 +141,39 @@ async function action(options) {
 
     const moreArgs: string[] = [];
 
-    if (tag) {
-      // Assign custom tag in the publish command, so we don't accidentally publish as latest.
-      moreArgs.push('--tag', tag);
-    }
+    // Assign custom tag in the publish command, so we don't accidentally publish as latest.
+    moreArgs.push('--tag', npmPublishTag);
 
     // Publish to NPM registry
+    console.log('Executing command: npm publish ' + ['--access', 'public', ...moreArgs].join(' '));
```

Comment:
Brent's comment applies here as well — maybe we should indicate it's a dry run 😉